pkg/networkmanager/firewall.jsx

```diff
@@ -760,7 +761,24 @@ class ActivateZoneModal extends React.Component {
                         <FormHelperText isHidden={false}>{_("The cockpit service is automatically included")}</FormHelperText>
                     </FormGroup>
 
-                    <FormGroup label={ _("Interfaces") } hasNoPaddingTop isInline>
+                    <FormGroup label={ _("Interfaces") }
+                               hasNoPaddingTop
+                               isInline
+                               labelIcon={
+                                   <Popover aria-label={_("Inerfaces help")}
+                                     enableFlip
+                                     bodyContent={
+                                         <Flex direction={{ default: 'column' }}>
+                                             <FlexItem>{_("All traffic going through selected interfaces will respect this zone's settings.")}</FlexItem>
+                                             <FlexItem>{_("Each interface can be assigned only to one zone.")}</FlexItem>
```

Comment:
> Also, this implies incoming and outgoing traffic, which isn't the case.. right?

I am not certain. Surely controlling incoming traffic is the most common usecase but I think that you can block also outgoing traffic with firewall. Not sure though if you can specify it in zones/services.

Reply:
> I am not certain. Surely controlling incoming traffic is the most common usecase but I think that you can block also outgoing traffic with firewall. Not sure though if you can specify it in zones/services.

You cannot block outgoing traffic with Cockpit's firewall. It only handles incoming traffic.

I talked with @jelly about this and recently wrote current thoughts on how to handle it @ https://github.com/cockpit-project/cockpit/issues/9047#issuecomment-1026013963 with a bit of it covered in the mockups on the comment right before it @ https://github.com/cockpit-project/cockpit/issues/9047#issuecomment-1026005844.

(I did think a lot about it and wrote things about outgoing as well as incoming a long time ago, and I think I even did some mockups for it back then too — but the firewall page has changed since the initial drafts, so I needed to redo it for the current implementation of the firewall page.)

But the reason why we only block incoming traffic with the firewall page is that it's what's wanted on a server almost all of the time, as the software running on a server is almost always a few carefully chosen or custom written apps. Plus, it would mean we need someone to implement it. (We still don't have everything implemented that was designed and mocked up, even for just handling incoming traffic.)

Blocking outgoing is more useful for workstations, where there's a lot of software written by many different authors and so many are doing network connections these days... but Cockpit's for servers, not workstations; so we haven't implemented it yet (as it wouldn't be so useful in most cases).

If we say "all traffic", then we're implying that the firewall actually affects all traffic, including outgoing... and this isn't true.

Adding a zone on the system _may_ affect some outgoing traffic if there are additional rules in the zone for it, but we don't provide anything in the UI to actually show or add to or manage that at the moment.

Reply:
To unstick this and move it forward, we could just drop the "all" and be intentionally vague (instead of trying to be specific):

```suggestion
                                             <FlexItem>{_("Traffic passing through selected interfaces will respect this zone's settings.")}</FlexItem>
                                             <FlexItem>{_("Each interface can be assigned only to one zone.")}</FlexItem>
```

It's still a little problematic, implying all but not outright saying it.

Once we land https://github.com/cockpit-project/cockpit/pull/17084 with some fixes, it should be clearer that the firewall is only for incoming traffic (for now at least).

Reply:
Changed it to `Traffic passing through...` - as it is correct (if the zone settings don't do anything about outgoing, it still respects these settings :)) and not too specific in the case that some zone or included service would actually affect outgoing traffic.